devicetypes/smartthings/zigbee-curtain.src/zigbee-curtain.groovy

```diff
@@ -14,7 +14,7 @@
 import physicalgraph.zigbee.zcl.DataType
 import physicalgraph.zigbee.clusters.iaszone.ZoneStatus
 metadata {
-    definition(name: "Zigbee Curtain", namespace: "smartthings", author: "SmartThings", ocfDeviceType: "oic.d.blind", mnmn: "SmartThings", vid: "generic-curtaion") {
```

Comment:
you had a typo here in `curtaion`. it should have been `curtain`

perhaps that's why you were not seeing it work correctly?

Reply:
This is because my colleague created a  `vid: generic-curtaion` uimetadata to `sideloading` test for me. I have deleted it in the next commit.